FhirToDataLake/deploy/templates/DeployContainerApp.json

```diff
@@ -0,0 +1,384 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "pipelineName": {
+        "type": "string",
+        "minLength": 3,
+        "maxLength": 24,
+        "metadata": {
+            "description": "The name of the container instance that you wish to create."
+        }
+    },
+    "fhirServerUrl": {
+        "type": "string",
+        "metadata": {
+            "description": "The fhir server endpoint you wish to get data."
+        }
+    },
+    "authentication": {
+        "type": "bool",
+        "defaultValue": false,
+        "allowedValues": [
+        true,
+        false
+        ],
+        "metadata": {
+            "description": "Whether to access Azure API for FHIR with managed identity authentication. Set false if you are using an OSS public FHIR server instance."
+        }
+    },
+    "fhirVersion": {
+        "type": "string",
+        "defaultValue": "R4",
+        "allowedValues": ["R4"]
+    },
+    "dataStart": {
+        "type": "string",
+        "defaultValue": "1970-01-01 00:00:00 +00:00",
+        "metadata": {
+            "description": "Start timestamp of the data range you want to export."
+        }
+    },
+    "dataEnd": {
+        "type": "string",
+        "defaultValue": "",
+        "metadata": {
+            "description": "End timestamp of the data range you want to export. Will continuous export all data if not specified."
+        }
+    },
+    "containerName": {
+        "type": "string",
+        "defaultValue": "fhir",
+        "metadata": {
+            "description": "The name of the container to store job and data."
+        }
+    },
+    "jobConcurrency": {
```

Comment:
the concurrent sets to 10 now?

Reply:
Set concurrency to 3 pre offline discussion.